charts/datadog/templates/_container-host-release-volumemounts.yaml

```diff
@@ -1,4 +1,5 @@
 {{- define "linux-container-host-release-volumemounts" -}}
+{{- if (not .Values.datadog.disableDefaultOsReleasePaths) }}
```

Comment:
Hi @faelis 

Instead of removing the `os-release-file`, does Talos.dev provide a similar file? by default we are trying to mount `/etc/os-release`.  If this file doesn't exist on talos.dev, could you check if an equivalent file exist? 

I'm thinking that maybe instead of add another option to disable the defaultOsReleasePaths, we could modify the current logic to check if `.Values.datadog.osReleasePath` is not empty. 
And in the v`alues.yaml` file set `datadog.osReleasePath: ""` WDYT?

Reply:
Ok so I was able to test it locally, And I can find the `/etc/os-release` file on a talos node

```
root@talos-default-worker-1 / [talos-default-worker-1]
docker > cat /etc/os-release
NAME="Talos"
ID=talos
VERSION_ID=v1.8.3
PRETTY_NAME="Talos (v1.8.3)"
HOME_URL="https://www.talos.dev/"
BUG_REPORT_URL="https://github.com/siderolabs/talos/issues"
VENDOR_NAME="Sidero Labs"
VENDOR_URL="https://www.siderolabs.com/"
```

Reply:
Hi, just checked on my side, I have the file in the last version too.

Maybe we could keep the feature in case of an OS (other than talos) that don't have the file?